.github/workflows/support-lambdas-build.yml

```diff
@@ -47,18 +47,12 @@ jobs:
           role-to-assume: ${{ secrets.GU_RIFF_RAFF_ROLE_ARN }}
           aws-region: eu-west-1
 
-      - name: Setup Java 11
+      - name: Setup Java
         uses: actions/setup-java@v4
         with:
-          java-version: "11"
```

Comment:
support lambdas includes these:

    `stripe-intent`,
    `it-test-runner`,
    `acquisitions-firehose-transformer`,
    `acquisition-events-api`,
    `bigquery-acquisitions-publisher`,

Reply:
I noticed that `acquisition-events-api` is getting built and deployed twice on every merge!  
<img width="242" alt="image" src="https://github.com/guardian/support-frontend/assets/7304387/a01ddfed-31fe-43a2-ae5c-c456f8c0faf4">

Once as part of this build and once in its own build.  I've deleted the dedicated build (at the top of this PR).

https://riffraff.gutools.co.uk/deployment/history?projectName=support%3Alambdas%3Aacquisition-events-api&stage=PROD&pageSize=20&page=1